build.sbt

```diff
@@ -5,13 +5,15 @@ import scala.sys.process._
 
 skip in publish := true
 
-lazy val integrationTestSettings: Seq[Def.Setting[_]] = Defaults.itSettings ++ Seq(
+lazy val integrationTestSettings: Seq[Def.Setting[_]] = Project.inConfig(IntegrationTest)(baseAssemblySettings) ++ Defaults.itSettings ++ Seq(
   scalaSource in IntegrationTest := baseDirectory.value / "src" / "test" / "scala",
   javaSource in IntegrationTest := baseDirectory.value / "src" / "test" / "java",
   resourceDirectory in IntegrationTest := baseDirectory.value / "src" / "test" / "resources",
   testOptions in Test += Tests.Argument(TestFrameworks.ScalaTest, "-l", "com.gu.test.tags.annotations.IntegrationTest"),
   testOptions in Test += Tests.Argument(TestFrameworks.ScalaTest, "-eU"),
-  testOptions in IntegrationTest += Tests.Argument(TestFrameworks.ScalaTest, "-eU")
+  testOptions in IntegrationTest += Tests.Argument(TestFrameworks.ScalaTest, "-eU"),
+  test in (IntegrationTest, assembly) := {},
+  aggregate in (IntegrationTest, assembly) := false
```

Comment:
- we don't want to run the tests when wer'e building them to run in AWS.
- we don't want to assembly a jar for every single project (yet) just for the main one.